lib/_tls_wrap.js

```diff
@@ -19,7 +19,7 @@ const defaultSessionIdContext = getDefaultSessionIdContext();
 function getDefaultSessionIdContext() {
   var defaultText = process.argv.join(' ');
   /* SSL_MAX_SID_CTX_LENGTH is 128 bits */
-  if (process.config.variables.openssl_fips) {
+  if (process.config && process.config.variables && process.config.variables.openssl_fips) {
```

Comment:
I generally wrap long lines myself but I decided to go with your suggestion from the previous PR:

https://github.com/nodejs/node/pull/6110#discussion_r58977126